test/framework/easyconfigs/yeb/sqlite-3.8.10.2.yeb

```diff
@@ -0,0 +1,31 @@
+_internal_variables_:
+    - &versionstr 3081002
```

Comment:
this is now hardcoding the version in two places, just in a different way, which we should try to avoid...

doesn't look trivial to change though, I must admit